### PR TITLE
Add `--fresh` option to remove scaffolding

### PR DESCRIPTION
**Pull request by davidhemphill**
I'm not a fan of the default scaffolding, but recognize other people are, and I don't hold it against them, so I'd rather pass in a `--fresh` flag when creating a new project. That's just me. No judgement here. Actually, the auth scaffolding is pretty handy, I just don't prefer Bootstrap. Different strokes.

### CODE REVIEW

File: laravel [hunk 1]
```diff
@@ -4,7 +4,7 @@
 require __DIR__.'/../../autoload.php';
 //require __DIR__.'/vendor/autoload.php';
 
-$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.0');
+$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.1');
```
Comment (GrahamCampbell) [inline laravel]: Bumping the version should not be done in pulls.

Comment (GrahamCampbell) [inline laravel]: Taylor will do this when he wants to do the next release.

Comment (davidhemphill) [inline laravel]: Gotcha, I'll remove that and force it up

**Comment by taylorotwell** [comment]
Gonna hold on this until it becomes clear what the included assets stuff will look like in 5.1.

**Comment by qw3rtman** [comment]
:+1: on this idea. Would be a great addition, I know I would use it.

**Pull request closed without merging**